### DISCUSSION TITLE
Found 2 archives with the id default

### DISCUSSION THREAD

**Discussion by madrigal1** [goreleaser/goreleaser#2041] [Q&A] [answered]
**Describe the bug**

![image](https://user-images.githubusercontent.com/30615210/106567121-48085d80-6557-11eb-83d4-488b34f21272.png)

**To Reproduce**

![image](https://user-images.githubusercontent.com/30615210/106567172-61110e80-6557-11eb-97be-363312ae064c.png)

> Make sure your config file is valid by running
> `goreleaser check -f path-to-config-file`.

**Expected behavior**

**Environment (please complete the following information):**

- OS: [e.g. mac, linux]
- OS version: [uname -a]
- GoReleaser Version [goreleaser --version]
- GitLab version (if using a private hosted one)

**Additional context**
how do i solve this error nothing is working.
i want to be able to change the binary's name

**Comment by caarlos0** [comment, marked as answer]
You have two archives in your config without specifying `id` for any of them... (one has only the `name_template` and the other only `replacements`)

The `id` property default is `default`, so you have 2 archives with the same id, as the error say. 

You'll need to fix your config file.

**Reply by madrigal1** [reply]
i am sorry i am bit of a beginner, can you suggest what fixes should be made in the config file  to resolve this issue?
adding the binary field in the builds section and using name templates tresults in : error=found 2 builds with the ID 'acmInternal-cli', please fix your config. any tips as to what i am doing wrong ?

**Reply by madrigal1** [reply]
ok have figured it out thanks leaving the yaml file for future reference by others:

![image](https://user-images.githubusercontent.com/30615210/106700264-7eeb7b80-660a-11eb-8e87-87cc1a848c0a.png)

**Reply by caarlos0** [reply]
I suggest you take a look at the yaml syntax for arrays, e.g. https://www.w3schools.io/file/yaml-arrays/

**Comment by madrigal1** [comment]
Thanks
